src/Forms/GridField/GridFieldAddExistingAutocompleter.php

```diff
@@ -136,7 +136,7 @@ public function getHTMLFragments($gridField)
 
         // If an object is not found, disable the action
         if (!is_int($gridField->State->GridFieldAddRelation(null))) {
-            $addAction->setReadonly(true);
+            $addAction->setDisabled(true);
```

Comment:
Is "use button tag" true by default? Otherwise I guess the `input` rendered by a form action could be readonly...

Reply:
Just had a look. [GridField_FormAction](https://github.com/silverstripe/silverstripe-framework/blob/5d7c5ffb07fe9e4ae35b5766a8b3b4460a485ca5/src/Forms/GridField/GridField_FormAction.php#L45) defaults to using a `<button>` tag.